### PR TITLE
add E2E test to change language feature

### LINKED ISSUE #1386: Add E2E test to change language feature
**Component**: ui

**Why this is needed**:
Because it's not tested yet

**What should be done**:
Add Cypress tests

**Implementation proposal** (strongly recommended):

**Test plan**:

### PR DESCRIPTION
**Pull request by ChengYanJin**
**Component**:
ui

**Context**: 
Add E2E test to change the language.

**Summary**:

**Acceptance criteria**: 
`./node_modules/cypress/bin/cypress run -b chrome -s cypress/integration/EditLanguage.feature -e userName=admin,password=admin --no-exit`

In this test, we first set the language of localstorage as English.
And then change the language into French.

Closes: #1386

### CODE REVIEW

File: ui/cypress/integration/common/login.js [hunk 1]
```diff
@@ -19,4 +21,5 @@ Given('I log in', () => {
     'contain',
     userName
   );
+  cy.wait(delayAfter);
```
Comment (carlito-scality) [inline ui/cypress/integration/common/login.js]: Why do you need a delay?

File: ui/src/containers/Layout.js [hunk 2]
```diff
@@ -78,15 +78,17 @@ class Layout extends Component {
         onClick: () => {
           this.props.updateLanguage(FR_LANG);
         },
-        selected: this.props.language === FR_LANG
+        selected: this.props.language === FR_LANG,
+        'data-cy': 'fr'
```
Comment (carlito-scality) [inline ui/src/containers/Layout.js]: use FR_LANG instead of 'fr'

File: ui/src/containers/Layout.js [hunk 3]
```diff
@@ -78,15 +78,17 @@ class Layout extends Component {
...
       },
       {
         label: 'English',
         name: 'EN',
         onClick: () => {
           this.props.updateLanguage(EN_LANG);
         },
-        selected: this.props.language === EN_LANG
+        selected: this.props.language === EN_LANG,
+        'data-cy': 'en'
```
Comment (carlito-scality) [inline ui/src/containers/Layout.js]: use EN_LANG instead of 'en'

**Review by carlito-scality — CHANGES_REQUESTED** [review]

File: ui/cypress/integration/common/editLanguage.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('I change the language by click the dropdown in the narbar', () => {
+  window.localStorage.setItem('language', 'en');
+  cy.get('.sc-dropdown')
+    .eq(0)
+    .click();
+  cy.get('[data-cy=fr]').click();
+
+  cy.get('.sc-trigger-text').should('contain', 'FR');
```
Comment (carlito-scality) [inline ui/cypress/integration/common/editLanguage.js]: should use ``` cy.get('.sc-dropdown').eq(0).get('.sc-trigger-text')``` instead to be more precise.

**Comment by carlito-scality** [comment]
@bert-e reset

**Comment by ChengYanJin** [comment]
@bert-e approve

**Review by carlito-scality — APPROVED** [review]
LGTM

**Pull request merged**